packages/js-example-app/src/index.server.ts

```diff
@@ -0,0 +1,233 @@
+import { render } from '@lit-labs/ssr';
+import { html } from 'lit';
+import '@boiler/ui-library';
+
+export function* renderIndex() {
+  yield* render(html`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8" />
+        <title>Boiler | Vanilla JS Example App</title>
+        <link href="https://fonts.cdnfonts.com/css/8bit-wonder" rel="stylesheet" />
+        <link href="/packages/js-example-app/src/style.css" rel="stylesheet" />
+      </head>
+      <body>
+        <div class="logoWrapper">
+          <img src="/packages/js-example-app/src/assets/logo.svg" width="200" height="100" alt="boiler" />
+          <h2>Vanilla JS Example Application</h2>
+        </div>
+
+        <div class="contentWrapper">
+          <div class="componentWrapper">
+            <div class="component">
+              <p>Button Text</p>
+              <blr-button-text
+                theme="Light"
+                variant="cta"
+                size-variant="md"
+                label="Button"
+                has-icon="true"
+                icon-position="leading"
+                icon="blr360"
+                button-text-id="button-id"
+              ></blr-button-text>
+              <button id="toggleLoadingState">Toggle Loading State</button>
+              <button id="toggleDisabledState">Toggle Disabled State</button>
+            </div>
+
+            <div class="component">
+              <p>Icon Button</p>
+              <blr-button-icon
+                theme="Light"
+                variant="primary"
+                size-variant="md"
+                icon="blr360"
+                aria-label="Button Icon"
+                button-icon-id="buttonIconId"
+              >
+              </blr-button-icon>
+            </div>
+
+            <div class="component">
```

Comment:
Currently not working. Need to explore the reason.

Reply:
Noted in new [Issue](https://github.com/deven-org/boiler/issues/1122#issue-2348276750). See point **B)**